prime-router/settings/staging/0170-ucsd-onboarding-receiver-settings.yml

```diff
@@ -70,7 +70,7 @@
           customLengthHl7Fields: {}
       jurisdictionalFilter:
         - "(Bundle.entry.resource.ofType(MessageHeader).event.code = 'R01')"
-        - "Bundle.entry.resource.ofType(MessageHeader).destination.receiver.resolve().identifier.where(value = 'R797').exists() or Bundle.entry.resource.ofType(MessageHeader).destination.receiver.resolve().identifier.where(value = 'R508').exists()"
+        - "(Bundle.entry.resource.ofType(DiagnosticReport)[0].basedOn.resolve().requester.resolve().organization.resolve().extension.where(url = 'https://reportstream.cdc.gov/fhir/StructureDefinition/xon-organization').extension.where(url = 'XON.10').value in ('R797' | 'R508'))"
```

Comment:
Should we also filter by sending facility (or application?). I realize it's unlikely another state would use 'R508' or 'R797' as receiving facility identifiers, but it would also be extremely bad if we misrouted results messages.

Reply:
That's a good idea. Do we know what values to use? The latest test files have `SISGDSP` for both sending facility and application

Reply:
As I recall, `SISGDSP` is the expected value for both MSH-3 and 4 for results coming from CDPH. I believe that's short for 'Screening Information System' & 'Genetic Disease Screening Program'. I wish that that indicated 'CA', or better yet was an OID, but it may be all we can key on.

Reply:
I updated it to add the filter for the sender